device-types/HPE/ProCurve-2910-48G.yaml

```diff
@@ -0,0 +1,112 @@
+---
+manufacturer: HPE
+model: ProCurve 2910al-48G
+slug: hpe-procurve-2910al-48g
+part_number: J9147A
+u_height: 1
+comments: '[ProCurve 2910al-48G Specs](https://support.hpe.com/hpesc/public/docDisplay?docId=emr_na-c01844343)'
+is_full_depth: false
+console-ports:
```

Comment:
Please add `airflow` information if you have it

Reply:
Do not have it, but tracking down the information. I'll update the PR when I get confirmation.

Reply:
Thanks, I will mark this as a draft. Once it is ready please mark it for review and ping me!